extensions/amp-date-display/0.2/test/test-amp-date-display.js

```diff
@@ -112,135 +109,20 @@ describes.realWin(
-      expect(data.day).to.equal('3');
-      expect(data.dayTwoDigit).to.equal('03');
-      expect(data.dayName).to.equal('Saturday');
-      expect(data.dayNameShort).to.equal('Sat');
-      expect(data.hour).to.equal('4');
-      expect(data.hourTwoDigit).to.equal('04');
-      expect(data.hour12).to.equal('4');
-      expect(data.hour12TwoDigit).to.equal('04');
-      expect(data.minute).to.equal('5');
-      expect(data.minuteTwoDigit).to.equal('05');
-      expect(data.second).to.equal('6');
-      expect(data.secondTwoDigit).to.equal('06');
-      expect(data.dayPeriod).to.equal('am');
-    });
-
-    describe('correctly parses', () => {
-      it('now keyword', async () => {
-        element.setAttribute('datetime', 'now');
-        element.build();
-
-        const {iso} = await getRenderedData();
-        const dateFromParsed = new Date(iso);
-
-        // Because of the runtime there could be a several ms difference.
-        expect(dateFromParsed.getTime()).to.equal(Date.now());
-      });
-
-      it('day only ISO 8601 date', async () => {
-        element.setAttribute('datetime', '2001-02-03');
-        element.build();
-
-        const data = await getRenderedData();
-
-        expect(data.iso).to.equal('2001-02-03T00:00:00.000Z');
-      });
-
-      it('full ISO 8601 date in UTC time zone', async () => {
-        element.setAttribute('datetime', '2001-02-03T04:05:06.007Z');
-        element.build();
-
-        const data = await getRenderedData();
-
-        expect(data.iso).to.equal('2001-02-03T04:05:06.007Z');
-      });
-
-      it('full ISO 8601 date without time zone (interpreted as local)', async () => {
-        element.setAttribute('datetime', '2001-02-03T04:05:06.007');
-        element.build();
-
-        const data = await getRenderedData();
-        const result =
-          `${data.year}-${data.monthTwoDigit}-${data.dayTwoDigit}` +
-          `T${data.hourTwoDigit}:${data.minuteTwoDigit}:${data.secondTwoDigit}`;
-
-        expect(result).to.equal('2001-02-03T04:05:06');
-      });
-
-      it('full ISO 8601 date in a custom time zone', async () => {
-        element.setAttribute('datetime', '2001-02-03T04:05:06.007+08:00');
-        element.build();
-
-        const data = await getRenderedData();
-
-        expect(data.iso).to.equal('2001-02-02T20:05:06.007Z');
-      });
-
-      it('seconds since the UNIX epoch', async () => {
-        element.setAttribute('timestamp-seconds', '981173106');
-        element.build();
-
-        const data = await getRenderedData();
-
-        expect(data.iso).to.equal('2001-02-03T04:05:06.000Z');
-      });
-
-      it('miliseconds since the UNIX epoch', async () => {
-        element.setAttribute('timestamp-ms', '981173106007');
-        element.build();
-
-        const data = await getRenderedData();
-
-        expect(data.iso).to.equal('2001-02-03T04:05:06.007Z');
-      });
-    });
-
-    it('adds offset seconds', async () => {
+    it('does not rerender', async () => {
```

Comment:
This is because `amp-date-display` doesn't interact with `amp-bind` at all, right? Why is this needed? Custom elements that don't use a callback typically also don't explicitly check that it's ineffective if called.

Reply:
It's to keep the same API as the original `amp-date-display`. It only uses `buildCallback`, which is just the first "render" called here.